From: Jan Engelhardt <jengelh@medozas.de>
Date: Sat, 3 Sep 2011 11:35:53 +0000 (+0200)
Subject: libxt_RATEEST: link with -lm
X-Git-Tag: v1.4.12.2~14^2~3
X-Git-Url: http://git.netfilter.org/cgi-bin/gitweb.cgi?p=iptables.git;a=commitdiff_plain;h=9249ad37b2342eb48009e18f3982362e1018ea5a

libxt_RATEEST: link with -lm

$ ldd -r libxt_RATEEST.so
undefined symbol: log   (./libxt_RATEEST.so)

Signed-off-by: Jan Engelhardt <jengelh@medozas.de>
---

diff --git a/extensions/GNUmakefile.in b/extensions/GNUmakefile.in
index dbf210c..107c9d5 100644
--- a/extensions/GNUmakefile.in
+++ b/extensions/GNUmakefile.in
@@ -96,6 +96,7 @@ lib%.oo: ${srcdir}/lib%.c
 	${AM_VERBOSE_CC} ${CC} ${AM_CPPFLAGS} ${AM_DEPFLAGS} ${AM_CFLAGS} -D_INIT=lib$*_init -DPIC -fPIC ${CFLAGS} -o $@ -c $<;
 
 # Need the LIBADDs in iptables/Makefile.am too for libxtables_la_LIBADD
+xt_RATEEST_LIBADD   = -lm
 xt_statistic_LIBADD = -lm
 
 

